src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -763,27 +851,33 @@ private void ParseHelper(string[] args)
                     _showHelp = true;
                     _showExtendedHelp = true;
                     _abortStartup = true;
+                    ParametersUsed |= ParameterBitmap.Help;
                 }
                 else if (MatchSwitch(switchKey, "login", "l"))
                 {
                     // On Windows, '-Login' does nothing.
                     // On *nix, '-Login' is already handled much earlier to improve startup performance, so we do nothing here.
+                    ParametersUsed |= ParameterBitmap.Login;
                 }
                 else if (MatchSwitch(switchKey, "noexit", "noe"))
                 {
                     _noExit = true;
+                    ParametersUsed |= ParameterBitmap.NoExit;
                     noexitSeen = true;
                 }
                 else if (MatchSwitch(switchKey, "noprofile", "nop"))
                 {
+                    ParametersUsed |= ParameterBitmap.NoProfile;
                     _skipUserInit = true;
                 }
                 else if (MatchSwitch(switchKey, "nologo", "nol"))
                 {
+                    ParametersUsed |= ParameterBitmap.NoLogo;
                     _showBanner = false;
                 }
                 else if (MatchSwitch(switchKey, "noninteractive", "noni"))
                 {
+                    ParametersUsed |= ParameterBitmap.NonInteractive;
                     _noInteractive = true;
                 }
                 else if (MatchSwitch(switchKey, "socketservermode", "so"))
```

Comment:
Is there a reason not to cover the other server modes?

Reply:
they're covered in other parts of the telemetry (see ConsoleHost line:195)

Reply:
I saw that later, but you cover the ssh server which is also covered later